Sources/Authenticator/Views/ContinueSignInWithTOTPSetupView.swift

```diff
@@ -0,0 +1,175 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import SwiftUI
+import CoreImage.CIFilterBuiltins
+
+
+/// Represents the content being displayed when the ``Authenticator`` is in the ``AuthenticatorStep/continueSignInWithTOTPSetup`` step.
+public struct ContinueSignInWithTOTPSetupView<Header: View,
+                                              Footer: View>: View {
+    @Environment(\.authenticatorState) private var authenticatorState
+    @Environment(\.authenticatorTheme) private var theme
+    @ObservedObject private var state: ConfirmSignInWithCodeState
+    @StateObject private var codeValidator: Validator
+    private let issuer: String?
+    private let headerContent: Header
+    private let footerContent: Footer
+
+    /// Creates a `ConfirmSignInWithTOTPView`
+    /// - Parameter state: The ``ConfirmSignInWithCodeState`` that is observed by this view
+    /// - Parameter headerContent: The content displayed above the fields. Defaults to  ``ContinueSignInWithTOTPSetupHeader``
+    /// - Parameter footerContent: The content displayed bellow the fields. Defaults to  ``ContinueSignInWithTOTPSetupFooter``
+    public init(
+        state: ConfirmSignInWithCodeState,
+        issuer: String?,
```

Comment:
If having the issuer in the state is not an option, another alternative would be for this argument to be of type `TOTPOptions?` instead of `String?`.

That would also make it more clear as to what to provide on the Authenticator's `continueSignInWithTOTPSetupContent` callback.